main.py

```diff
@@ -163,7 +163,7 @@ async def eat(message):
     profile, _ = Profile.get_or_create(guild_id=message.guild.id, user_id=message.user.id)
     profile.tokens -= 1
     profile.save()
-    await message.response.send_message("You eat a coal. Yum!")
+    await message.response.send_message(f"You eat a coal. Yum!\nYou now have {profile.tokens} coal remaining.")
```

Comment:
maybe "tokens remaining"? it would be more consistent with the terminology (although it does say you are eating a coal not a token but i think thats fine)

Reply:
done